BSP/Inc/BSP_ADC.h

```diff
@@ -0,0 +1,10 @@
+#include "common.h"
+#include "stm32f4xx_adc.h"
+#include "config.h"
+#include "stm32f4xx.h"
+
+//Initialize ADC
```

Comment:
Can we comment the functions with the same comment style as in `BSP/Inc/BSP_CAN.h`?

Reply:
Can you fix up the comments in this file @tasniabari ?